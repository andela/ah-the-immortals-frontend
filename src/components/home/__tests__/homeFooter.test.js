import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import Footer from '../homeFooter';

describe('Tests home footer', () => {
  it('tests footer component', () => {
    const wrapper = shallow(
      <Footer />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
