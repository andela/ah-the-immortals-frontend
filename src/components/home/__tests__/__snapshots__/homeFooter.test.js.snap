// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests home footer tests footer component 1`] = `
<footer
  className="bg-light text-dark mt-4 fixed-bottom"
>
  <div
    className="container-fluid py-2"
  >
    <div
      className="row"
    >
      <div
        className="col-md-3 footmargin"
      >
        Authors Haven
         
        <span
          className="small"
        >
          <br />
          <a
            href="#null"
          >
            Terms and Conditions
          </a>
        </span>
      </div>
      <div
        className="col-md-8 text-right small align-self-end"
      >
        Copyright Â©
         
        2019
         
        Immortals
      </div>
    </div>
  </div>
</footer>
`;
